src/twisted/mail/pop3.py

```diff
@@ -36,7 +36,7 @@
 from twisted import cred
 
 ##
-## Authentication
+#  Authentication
```

Comment:
Probably should leave the comment style alone unless you want to make a case for  changing it throughout the module.